### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [r-value/hugo-blog#54] [Announcements]
# about/

关于 rvalue 退役 OIer / 车万众 / 音游玩家 / CS 在读 / 本科 2020 级 喜欢计算机, 啥都想学 然后啥都学不会 关于本博客 OI 时代的文章从 cnblogs 迁移而来 主要是因为 cnblogs 出现了不稳定

https://blog.rvalue.moe/about/

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。
